tests/model_serving/model_runtime/vllm/utils.py

```diff
@@ -88,8 +89,14 @@ def fetch_tgis_response(  # type: ignore
     return model_info, completion_responses, stream_completion_responses
 
 
+@retry(stop=stop_after_attempt(5), wait=wait_exponential(min=1, max=6))
```

Comment:
please use `retry` from TimeoutSampler

Reply:
I don't want to use that. I think the intensity here is good for my work, and I want to remain consistent with the vLLM util, so I'll stick with this retry.